src/js/hca-rjsf/definitions/child.js

```diff
@@ -0,0 +1,87 @@
+import _ from 'lodash/fp';
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+import { uiSchema as fullNameUI } from '../../common/schemaform/definitions/fullName';
+import { uiSchema as ssnUI } from '../../common/schemaform/definitions/ssn';
+
+
+export const schema = _.merge(_.omit(
```

Comment:
Yeah, I was thinking it'd be silly to do that unless VA comes out with another health care application (please no!), but I can do that for consistency.